Description: fix typos
Author: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Forwarded: https://github.com/s3tools/s3cmd/pull/504
Last-Update: 2015-03-18

--- s3cmd-1.5.2.orig/NEWS
+++ s3cmd-1.5.2/NEWS
@@ -197,7 +197,7 @@ Minor features and bugfixes:
 * Don't print nasty backtrace on KeyboardInterrupt.
 * Should work fine on non-UTF8 systems, provided all
   the files are in current system encoding.
-* System encoding can be overriden using --encoding.
+* System encoding can be overridden using --encoding.
 * Improved resistance to communication errors (Connection 
   reset by peer, etc.)
 
--- s3cmd-1.5.2.orig/S3/Exceptions.py
+++ s3cmd-1.5.2/S3/Exceptions.py
@@ -26,7 +26,7 @@ class S3Exception(Exception):
 
     def __str__(self):
         ## Call unicode(self) instead of self.message because
-        ## __unicode__() method could be overriden in subclasses!
+        ## __unicode__() method could be overridden in subclasses!
         return deunicodise(unicode(self))
 
     def __unicode__(self):
--- s3cmd-1.5.2.orig/s3cmd
+++ s3cmd-1.5.2/s3cmd
@@ -1989,7 +1989,7 @@ def get_commands_list():
     {"cmd":"rm", "label":"Delete file from bucket (alias for del)", "param":"s3://BUCKET/OBJECT", "func":cmd_object_del, "argc":1},
     #{"cmd":"mkdir", "label":"Make a virtual S3 directory", "param":"s3://BUCKET/path/to/dir", "func":cmd_mkdir, "argc":1},
     {"cmd":"restore", "label":"Restore file from Glacier storage", "param":"s3://BUCKET/OBJECT", "func":cmd_object_restore, "argc":1},
-    {"cmd":"sync", "label":"Synchronize a directory tree to S3 (checks files freshness using size and md5 checksum, unless overriden by options, see below)", "param":"LOCAL_DIR s3://BUCKET[/PREFIX] or s3://BUCKET[/PREFIX] LOCAL_DIR", "func":cmd_sync, "argc":2},
+    {"cmd":"sync", "label":"Synchronize a directory tree to S3 (checks files freshness using size and md5 checksum, unless overridden by options, see below)", "param":"LOCAL_DIR s3://BUCKET[/PREFIX] or s3://BUCKET[/PREFIX] LOCAL_DIR", "func":cmd_sync, "argc":2},
     {"cmd":"du", "label":"Disk usage by buckets", "param":"[s3://BUCKET[/PREFIX]]", "func":cmd_du, "argc":0},
     {"cmd":"info", "label":"Get various information about Buckets or Files", "param":"s3://BUCKET[/OBJECT]", "func":cmd_info, "argc":1},
     {"cmd":"cp", "label":"Copy object", "param":"s3://BUCKET1/OBJECT1 s3://BUCKET2[/OBJECT2]", "func":cmd_cp, "argc":2},
@@ -2295,7 +2295,7 @@ def main():
     logging.root.setLevel(cfg.verbosity)
 
     ## Default to --progress on TTY devices, --no-progress elsewhere
-    ## Can be overriden by actual --(no-)progress parameter
+    ## Can be overridden by actual --(no-)progress parameter
     cfg.update_option('progress_meter', sys.stdout.isatty())
 
     ## Unsupported features on Win32 platform
--- s3cmd-1.5.2.orig/s3cmd.1
+++ s3cmd-1.5.2/s3cmd.1
@@ -50,7 +50,7 @@ s3cmd \fBrestore\fR \fIs3://BUCKET/OBJEC
 Restore file from Glacier storage
 .TP
 s3cmd \fBsync\fR \fILOCAL_DIR s3://BUCKET[/PREFIX] or s3://BUCKET[/PREFIX] LOCAL_DIR\fR
-Synchronize a directory tree to S3 (checks files freshness using size and md5 checksum, unless overriden by options, see below)
+Synchronize a directory tree to S3 (checks files freshness using size and md5 checksum, unless overridden by options, see below)
 .TP
 s3cmd \fBdu\fR \fI[s3://BUCKET[/PREFIX]]\fR
 Disk usage by buckets
